python/cugraph/centrality/betweenness_centrality.pxd

```diff
@@ -19,18 +19,22 @@
 from cugraph.structure.graph_new cimport *
 from libcpp cimport bool
 
+cdef extern from "raft/handle.hpp" namespace "raft":
```

Comment:
Dont need to expose handle, it should already be cimported from graph_new.pxd

Reply:
Removed in 09cbb8c